draftfast/optimizer.py

```diff
@@ -56,6 +52,25 @@ def solve(self):
         solution = self.solver.Solve()
         return solution == self.solver.OPTIMAL
 
+    def _set_player_constraints(self):
```

Comment:
Nice, like this refactor. One thing we should be mindful of is that for showdown, you may want the ability to ban a player as a captain but allow him to be picked for another spot. Not sure how to handle this in `Optimizer`, so let me know if you have ideas. It's also outside the scope of this PR so feel free to ignore this.